### ISSUE TITLE
Android Apk Icon not produced properly

### ISSUE THREAD

**Issue by HappyCoder256** [flet-dev/flet#4550]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

Am kind of experiencing an error when producing the Android Apk icon. My root directory already has a the assests folder consisting of the favicon.png image but in previous version, the icon was produced and cropped very well. I don't know if am the only one experiencing this. Below is the image of the Produced app icon when installed on Android;
Android Version: 10
Flet version: 0.25.1
Operating System: Linux
![Screenshot_20241212-164655_One UI Home](https://github.com/user-attachments/assets/96fe9afa-b6e4-427b-9444-6168ff87fab2)

### Code sample

from flet import *
import flet
def main(page : Page):
    page.padding = 0
    cine = Stack([
            Container(
                width=page.width,
                height=page.height,
                content=Image(
                    src=f"./assets/N9ini_special/n9ini.jpeg",
                    width=page.width,
                    height=page.height,
                    fit=ImageFit.COVER,
                ),expand=True
                ),
            Container(
                content=Column([
                    Row([
                        IconButton(
                            icon=Icons.ARROW_BACK_IOS_NEW_ROUNDED,  # Movies icon
                            icon_size=24,        # Icon size
                            tooltip="back",    # Tooltip
                            #on_click=julie_back
                        ),
                    ],
                    alignment=MainAxisAlignment.SPACE_BETWEEN,  # Add more spacing between the sections
                    #tight=True,  # Prevent the row from expanding and causing extra black space
                    #expand=True,
                    vertical_alignment=CrossAxisAlignment.CENTER,),
                    Container(
                        content=Container(
                            content=Column([Text("Search your favorite content", weight='bold', size=15.5,style=TextStyle(font_family="Montserrat"), color="white", text_align=TextAlign.CENTER, no_wrap=False), Text("BY YEAR", text_align=TextAlign.CENTER, no_wrap=False, size=14.5,style=TextStyle(font_family="Montserrat"), color="white")], spacing= 2,alignment= MainAxisAlignment.CENTER,horizontal_alignment=CrossAxisAlignment.CENTER),
                            width=10000,
                            height=(18/100)*page.height,
                            border_radius=15,
                            blur=Blur(100, 100, BlurTileMode.MIRROR),
                            expand=True,
                            #alignment="center"
                        ),
                        padding=10,
                        #alignment="center"
                    ),
                    Container(
                        content=Column([
                            Container(
                                padding=0,
                                border_radius=15,
                                blur=Blur(100, 100, BlurTileMode.MIRROR),
                                content=TextField(max_length=4,shift_enter=True,cursor_error_color="red",text_size=13.5,text_align=TextAlign.CENTER, cursor_color="blue",enable_interactive_selection=True,border_radius=15,focused_border_width=0,border_width=0,hint_text="Enter year of release here",text_style=TextStyle(font_family="Montserrat")))
                        ],alignment= MainAxisAlignment.CENTER,horizontal_alignment=CrossAxisAlignment.CENTER),
                        padding=20,
                    )
                ]),
                border_radius=8,
                blur=Blur(10, 10, BlurTileMode.MIRROR),
                width=page.width,
                height=page.height,
            )
    ])
    page.add(cine)
if __name__ == "__main__":
    #import logging
    #logging.basicConfig(level=logging.DEBUG)
    flet.app(target=main,assets_dir="assets")#(), host="192.168.43.204", port=5662,view=AppView.WEB_BROWSER)#,view=AppView.WEB_BROWSER

### To reproduce

Icon being produced in an unexpected way

### Screenshots / Videos

<details open>
<summary>Captures</summary>

[Upload media here]

</details>

### Operating System

Linux

### Operating system details

Linux

### Flet version

0.25.1

### Regression

Yes, it used to work in a previous Flet version (please specify the version in additional details)

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

**Comment by yucel685** [comment]
Yes, the icon was previously formed in a square shape without any problems. now it forms round and doesn't look nice

**Comment by HappyCoder256** [comment]
Any changes i could make to revert the icon production settings

**Comment by ndonkoHenri** [comment]
This issue should have been fixed in the latest versions.
Give it a try and let know.
